src/util/FactorUtil.js

```diff
@@ -129,6 +129,12 @@ function (Okta, TimeUtil) {
       description: Okta.loc('factor.customFactor.description', 'login'),
       iconClassName: 'mfa-custom-factor',
       sortOrder: 16
+    },
+    'CUSTOM_HOTP': {
+      label: '',
```

Comment:
Is this empty because enroll is through api?

Reply:
I might update it if required during enroll.